src/test/java/HomePage.java

```diff
@@ -0,0 +1,48 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+
+public class HomePage extends BasePage {
+
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
**Please, remove code redundancies**

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

Reply:
Removed extra wait.